stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/application/ChatInitializer.kt

```diff
@@ -37,6 +39,7 @@ import io.getstream.chat.ui.sample.feature.HostActivity
 
 class ChatInitializer(private val context: Context) {
 
+    @OptIn(InternalStreamChatApi::class)
```

Comment:
Why is it needed to use this annotation?
It is the sample app that should be what our customers need to implement.
They shouldn't use this annotation on their code